src/index.js

```diff
@@ -96,6 +105,12 @@ export {
   AxesFold,
   Button,
   SingleSidebarItem,
+  GraphCreatePanel,
+  StyleLayoutPanel,
+  StyleAxesPanel,
+  StyleLegendPanel,
+  StyleNotesPanel,
+  StyleTracesPanel,
 };
```

Comment:
Ah yeah, it's nice that they're exposed now so users can pick and chose the ones they'd like to use/replace

Reply:
should we add a note to the readme?

Reply:
Not really necessary at this point, I don't think, it's mostly just easier for us to work with :)